book/src/storage/uid-and-id.md

```diff
@@ -36,8 +36,8 @@ let uid = object::new(ctx);
 ```
 
 On Sui, `UID` acts as a representation of an object, and allows defining behaviors and features of
-an object. One of the key-features - [Dynamic Fields]() - is possible because of the `UID` type
-being explicit. Additionally, it allows the [Transfer To Object (TTO)]() which we will explain later
+an object. One of the key-features - [Dynamic Fields](https://docs.sui.io/concepts/dynamic-fields) - is possible because of the `UID` type
```

Comment:
```suggestion
an object. One of the key features - [Dynamic Fields](../programmability/dynamic-fields.html) - is possible because of the `UID` type
```
That page on sui.docs site will go away soon